masterlist.yaml

```
@@ -11766,6 +11766,13 @@ plugins:
       - crc: 0x23EB2E98
         util: 'SSEEdit v4.0.4'
 
+  - name: 'SeasonalWildlifeDistribution.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/63700/' ]
```

The next version of LOOT will be able to display locations in the UI, making use of location names for hyperlinks instead of using generic text, so it'd be beneficial to always add them going forward:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63700/'
        name: 'Seasonal Wildlife Distribution'
```